### DISCUSSION TITLE
Crash: At Prompt.Select, I can select 'Value is required' when list is empty

### DISCUSSION THREAD

**Discussion by ryanm8227** [shibayan/Sharprompt#206] [Ideas]
Hello,

Here is a code example:

`using Sharprompt;`
...
`var city = Prompt.Select("Select your city", new[] { "Seattle", "London", "Tokyo" })`
`Console.WriteLine($"Hello, {city}!");`

The problem is that when the app is running, the user writes something (to filter the select options) and the user presses enter, the `Value is required` message shows up and if the user presses the arrow key down and then enter, the app crashes. I believe it is because the `Value is required` message acts as a selectable option and when it disappears and the user presses enter, the selection of an item which is not present is attempted.

I have attached a video to aid the problem comprehension.

This might be a good problem to fix.

Thank you, Ryan.

https://user-images.githubusercontent.com/60076318/161445252-f5743ea6-0be7-4210-98ad-00fc7d4fcf14.mp4

**Comment by shibayan** [comment]
Thanks for the report. This is clearly a pagination bug and we fixed it in #209.
